lib/tether/netfilter/netfilter_linux.go

```diff
@@ -158,7 +160,7 @@ func (r *Rule) args() ([]string, error) {
 	return args, nil
 }
 
-func iptables(ctx context.Context, args []string) error {
+func iptables(ctx context.Context, args []string) (tether.UtilityFn, error) {
 	args = append(args, "-w")
```

Comment:
Removed given your updated on timing. We have a retry so should be functional viable in my proposed scenario.